index.bs

```diff
@@ -10467,177 +10467,195 @@ asynchronous sections can't be reordered.
-given thread can only be executed when no other [=Atomically|atomic=]
-operation is running on another thread.
+Operations that happen <dfn>atomically</dfn> on a given thread can only be
+executed when no other [=Atomically|atomic=] operation is running on another
+thread.
 
-The algorithm for rendering a block of audio from an
-{{AudioContext}} <em>G</em> with a <a href="#control-message-queue">control message queue</a> <em>Q</em> is as
-follows.
+The algorithm for rendering a block of audio from an {{AudioContext}} <em>G</em>
+with a <a href="#control-message-queue">control message queue</a> <em>Q</em> is
+comprised of 3 steps:
+	1. Process the control message queue.
+	2. Process a render quantum.
+	3. Perform a microtask check point.
 
-If the algorithm returns true then it MUST be executed again in the
-future, to render the next block of audio. Else, the <a>rendering
-thread</a> yields and the processing stops. The <a>control thread</a>
-can restart executing this algoritm if needed.
+The order of these steps cannot be altered or interrupted. If the second step
+returns true then it MUST be executed again in the future, to render the
+subsequent block of audio. Otherwise, the <a>rendering thread</a> yields and the
+processing stops. The <a>control thread</a> can restart executing these steps if
+needed.
 
 <div class="note">
-	In practice, the {{AudioContext}} <a>rendering thread</a> is
-	often running off a system level audio callback, that executes in
-	an isochronous fashion. This callback passes in a buffer that has
-	to be filled with the audio that will be output. The size of the
-	buffer is often larger than a rendering quantum. In this case,
-	multiple invocations of the rendering algorithm will be called in a
-	rapid succession, in the same callback, before returning. After
-	some time, the underlying audio system will call the callback
-	again, and the algorithm will be executed again. This is an
-	implementation detail that should not be observable, apart from the
-	latency implications.
-
-	{{OfflineAudioContext}} will execute the algorithm continuously,
-	until {{OfflineAudioContext/length}} frames have been rendered.
+	In practice, the {{AudioContext}} <a>rendering thread</a> is often running off
+	a system level audio callback, that executes in an isochronous fashion. This
+	callback passes in a buffer that has to be filled with the audio that will be
+	output. The size of the buffer is often larger than a rendering quantum. In
+	this case, multiple invocations of the rendering algorithm will be called in a
+	rapid succession, in the same callback, before returning. After some time, the
+	underlying audio system will call the callback again, and the algorithm will
+	be executed again. This is an implementation detail that should not be
+	observable, apart from the latency implications.
+
+	{{OfflineAudioContext}} will execute the algorithm continuously, until
+	{{OfflineAudioContext/length}} frames have been rendered.
```

Comment:
Did you really change this in any way other than wrapping lines?  (The preview diff shows no changes.)

Please don't do that, if so, and revert this change.